### PR TITLE
Add segment events for external links usage

### PR DESCRIPTION
**Pull request by VkoHov**
- Change demo write key
- Add tracking events on GitHub, Slack links

### CODE REVIEW

File: aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx [hunk 1]
```diff
@@ -118,6 +120,9 @@ function MetricsBar({
                 href='https://github.com/aimhubio/aim#searching-experiments'
```
Comment (roubkar) [inline aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx:120]: @gorarakelyan, what should be the current link for the `Searching Experiments` section in the docs, or do we need this link here at all?

Comment (gorarakelyan) [reply aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx:120]: QL basics @roubkar, but that is in progress

**Review by roubkar — APPROVED** [review]

File: aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx (hunk 1 above)
Comment (gorarakelyan) [reply aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx:120]: Added https://aimstack.readthedocs.io/en/latest/quick_start/QL_basics.html#searching-runs-and-metrics :)

Comment (roubkar) [reply aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx]: We're removing this link from the menu now, in the future will add a link to docs in sidebar

Comment (gorarakelyan) [reply aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx]: @roubkar ok

**Pull request merged**